src/Browser/Avalonia.Browser/build/Avalonia.Browser.targets

```diff
@@ -6,6 +6,7 @@
   <ItemGroup>
     <!-- So we can access Emscripten APIs -->
     <EmccExportedRuntimeMethod Include="GL" />
+    <EmccExportedRuntimeMethod Include="PThread" />
```

Comment:
Should it be included only when WasmEnableThreads is `true`?

Reply:
I would also expect it to be always included by the runtime itself, when threading is enabled

Reply:
But I suppose DllImport compilation fails, if we don't have this at any build.